### DISCUSSION TITLE
It there a way to run program from given AST?

### DISCUSSION THREAD

**Discussion by SchrodingerZhu** [oracle/graal#5594] [General] [answered]

**Comment by alina-yur** [comment]
hi @SchrodingerZhu, is this discussion related to https://github.com/oracle/graal/issues/5598? IF so can we close the discussion and keep talking about it in the issue?

**Comment by alina-yur** [comment, marked as answer]
I think this should cover it: https://github.com/oracle/graal/issues/5598#issuecomment-1354990232
